src/settings/mod.rs

```diff
@@ -358,8 +363,13 @@ impl Settings {
 
         match config.try_into::<Settings>() {
             Ok(settings) => {
-                if current_env == "production" && &settings.hmackey == "YOU MUST CHANGE ME" {
-                    panic!("Please set a valid HMAC key.")
+                if current_env == "production" {
+                    if &settings.hmackey == "YOU MUST CHANGE ME" {
+                        panic!("Please set a valid HMAC key.")
+                    }
+                    if &settings.secretkey == "youmustchangethisfortheproductionenvironment" {
```

Comment:
Would it make sense to use the same default value for both of these two settings so that we can re-use a constant in these checks? Fwiw `"YOU MUST CHANGE ME"` also appears in the configs for the auth and content servers so seems to be a fairly conventional choice for an FxA repo.

Reply:
It would and I did that at first, but this string has to be an exact number of characters or rocket doesn't build... That's I changed it to that weird one, hehe